src/ArticleList.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleItem from "./ArticleItem";
+import Footer from "./Footer";
+
+const ArticleList = ({ setNumArticles, pageNumber }) => {
+	const [isLoading, setIsLoading] = useState(true);
+	const [articles, setArticles] = useState([]);
+
+	useEffect(() => {
+		setIsLoading(true);
```

Comment:
Nice use of the `isLoading` pattern - consider extrating your api calls into a separate file. It will keep your components free of bloat later on!